crates/bevy_gltf/src/loader.rs

```diff
@@ -682,6 +669,23 @@ fn get_gltf_extras(extras: &gltf::json::Extras) -> Option<GltfExtras> {
     })
 }
 
+fn node_transform(node: &Node) -> Transform {
```

Comment:
We should comment about why we're doing our own computation here.

Reply:
I've added doc comment to `note_transform` function.